src/components/UnifiedWalletModal/index.tsx

```diff
@@ -377,6 +377,10 @@ const UnifiedWalletModal: React.FC<IUnifiedWalletModal> = ({ onClose }) => {
       <Header onClose={onClose} />
       <div tw="border-t-[1px] border-white/10" />
       <ListOfWallets list={list} onToggle={onToggle} isOpen={isOpen} />
+
+      {walletModal?.footer ? (
+        <div tw="sticky bottom-0 right-0 left-0 p-5 border-t border-white/10">{walletModal?.footer}</div>
```

Comment:
let's render as it, without the default container classes

`{walletModal.footer ? <>{walletModal.footer}</> : null}`